LetsDoStuff.Domain/Models/DTO/GetUserByUserNameQueryResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace LetsDoStuff.Domain.Models.DTO
+{
+    public sealed class GetUserByUserNameQueryResult
```

Comment:
Let's call it response. And what do you think of calling it like `UserResponse` or `UserInfoResponse`?